feature-pack/src/main/resources/modules/system/layers/base/org/wildfly/microprofile/opentracing-smallrye/main/module.xml

```diff
@@ -44,6 +47,8 @@
         <module name="io.smallrye.opentracing"/>
 
         <module name="org.apache.thrift"/>
+        <module name="org.eclipse.microprofile.opentracing"/>
         <module name="org.jboss.logging"/>
+        <module name="org.jboss.resteasy.resteasy-jaxrs"/>
```

Comment:
We probably should be using `javax.ws.rs.api` unless this explicitly requires RESTEasy.

Reply:
The class WildFlyClientTracingRegistrarProvider casts directly to `ResteasyClientBuilder`, as it's done on the TCK: https://github.com/smallrye/smallrye-opentracing/blob/master/tck/src/test/java/io/smallrye/opentracing/ResteasyClientTracingRegistrarProvider.java#L24

Reply:
Ah interesting. We'll stick with it then.